### PR TITLE
Editor Controller

### PR DESCRIPTION
**Pull request by JavaLeg**
Save System able to save the new State models into the core/asset/library dir via the Editor Controller.

### CODE REVIEW

File: LibGDXGradle/desktop/src/Interface/ImageID.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		super(region);
+		this.status = status;
+		this.row = row;
+		this.column = column;
+	}
+
+
+	//
+	public void setPosition(int x, int y) {
+		super.setPosition(x, y);
```
Comment (SamWiseKimchi) [inline LibGDXGradle/desktop/src/Interface/ImageID.java]: Do you need to implement these functions? they just seem to be calling the inherited functions.

File: LibGDXGradle/desktop/src/Interface/MainMenuScreen.java [hunk 2]
```diff
@@ -76,7 +78,12 @@ public void show() {
         playButton.addListener(new ClickListener(){
             @Override
             public void clicked(InputEvent event, float x, float y) {
-                ((Game)Gdx.app.getApplicationListener()).setScreen(new EditorScreen(game));
+                try {
+					((Game)Gdx.app.getApplicationListener()).setScreen(new EditorScreen(game));
```
Comment (SamWiseKimchi) [inline LibGDXGradle/desktop/src/Interface/MainMenuScreen.java]: indenting is off here

File: LibGDXGradle/desktop/src/Interface/ImageID.java (hunk 1 above)
Comment (JavaLeg) [reply LibGDXGradle/desktop/src/Interface/ImageID.java]: @MatthewC221

Comment (MatthewC221) [reply LibGDXGradle/desktop/src/Interface/ImageID.java]: I don't think I do, I was planning on doing something more with them

**Pull request merged**